pastis/temporal_analysis/harris_mode.ipynb

```diff
@@ -0,0 +1,3338 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "30a87e9f",
+   "metadata": {},
+   "source": [
+    "## This notebook contains all the scripts used to do segment level thermal sensitivity analysis and produce plots all plots required for presentation, report or article. The code block which are commented aren't necessary to do full e2e analysis, but are used to support report/article."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "9cb4388c",
+   "metadata": {},
+   "source": [
+    "###  Import neccessary python libraries and pastis functions"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "c90040ca",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import os\n",
+    "os.chdir(\"/Users/asahoo/repos/PASTIS\")\n",
+    "import time\n",
+    "from shutil import copy\n",
+    "from astropy.io import fits\n",
+    "import astropy.units as u\n",
+    "import hcipy\n",
+    "import numpy as np\n",
+    "import pastis.util as util    \n",
+    "from pastis.config import CONFIG_PASTIS \n",
+    "from pastis.e2e_simulators.luvoir_imaging import LuvoirA_APLC \n",
+    "from pastis.e2e_simulators.generic_segmented_telescopes import SegmentedAPLC\n",
+    "import matplotlib.pyplot as plt\n",
+    "import pandas as pd\n",
+    "from scipy.interpolate import griddata\n",
+    "import exoscene.image\n",
+    "import exoscene.star\n",
+    "import exoscene.planet\n",
+    "from exoscene.planet import Planet\n",
+    "from astropy.io import fits as pf\n",
+    "from matplotlib.colors import TwoSlopeNorm\n",
+    "import matplotlib.gridspec as gridspec\n",
+    "from pastis.temporal_analysis.close_loop_analysis import req_closedloop_calc_batch"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "f1f4e410",
+   "metadata": {},
+   "source": [
+    "### Read from config file"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "34fcd7e7",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "nb_seg = CONFIG_PASTIS.getint('LUVOIR', 'nb_subapertures')\n",
+    "wvln = CONFIG_PASTIS.getfloat('LUVOIR', 'lambda') * 1e-9  # m #this doesn't matter, luvoir.wvln\n",
```

Comment:
this is not necessary indeed, please replace all instances of `wvln` with `luvoir.wvln`, that way everything that belongs together is tied up together.